docs/UserGuide.md

```diff
@@ -10,29 +10,30 @@
-        + [3.3.3.2 Deleting an attendance record for a student: `attendance delete`](#3332-deleting-an-attendance-record-for-a-student-attendance-delete)
+    + [3.2.1 Adding a student: `add` (By: Alex and Hogan)](#321-adding-a-student---add---by--alex-and-hogan-)
+    + [3.2.2 Editing a student: `edit` (By: Vaishak)](#322-editing-a-student---edit---by--vaishak-)
+    + [3.2.3 Locating students: `find` (By: Choon Siong)](#323-locating-students---find---by--choon-siong-)
+    + [3.2.4 Listing all students: `list`](#324-listing-all-students---list-)
+    + [3.2.5 Deleting a student: `delete`](#325-deleting-a-student---delete-)
+    + [3.2.6 Sorting the list of students: `sort` (By: Choon Siong)](#326-sorting-the-list-of-students---sort---by--choon-siong-)
+    + [3.2.7 Finding students with overdue fees: `overdue` (By: Ying Gao)](#327-finding-students-with-overdue-fees---overdue---by--ying-gao-)
+    + [3.2.8 Managing details for a student: `detail` (By: Vaishak)](#328-managing-details-for-a-student---detail---by--vaishak-)
+        + [3.2.8.1 Adding a detail: `detail add`](#3281-adding-a-detail---detail-add-)
+        + [3.2.8.2 Editing a detail: `detail edit`](#3282-editing-a-detail---detail-edit-)
+        + [3.2.8.3 Deleting a detail: `detail delete`](#3283-deleting-a-detail---detail-delete-)
+    + [3.2.9 Clearing all entries: `clear`](#328-clearing-all-entries---clear-)
+  * [3.3 Student Academics Features](#33-stude9t-academics-features)
+    + [3.3.1 Recording questions from a student: `question` (By: Ying Gao)](#331-recording-questions-from-a-student---question---by--ying-gao-)
+        + [3.3.1.1 Adding a question: `question add`](#3311-adding-a-question---question-add-)
+        + [3.3.1.2 Resolving a question: `question solve`](#3312-resolving-a-question---question-solve-)
+        + [3.3.1.3 Deleting a question: `question delete`](#3313-deleting-a-question---question-delete-)
+    + [3.3.2 Recording exams of a student: `exam` (By: Hogan)](#332-recording-exams-of-a-student---exam---by--hogan-)
+        + [3.3.2.1 Adding an exam record to a student: `exam add`](#3321-adding-an-exam-record-to-a-student---exam-add-)
+        + [3.3.2.2 Deleting an exam record for a student: `exam delete`](#3322-deleting-an-exam-record-for-a-student---exam-delete-)
+        + [3.3.2.3 Viewing exam statistics of a student: `exam stats`](#3323-viewing-exam-statistics-of-a-student---exam-stats-)
+    + [3.3.3 Recording attendance of a student: `attendance` (By: Vaishak)](#333-recording-attendance-of-a-student---attendance---by--vaishak-)
+        + [3.3.3.1 Adding an attendance record to a student: `attendance add`](#3331-adding-an-attendance-record-to-a-student---attendance-add-)
+        + [3.3.3.2 Deleting an attendance record for a student: `attendance delete`](#3332-deleting-an-attendance-record-for-a-student---attendance-delete-)
```

Comment:
Why are these changed uh, cos i think the extra `---` in the link breaks the links.

Reply:
Oh yea my bad. Not sure what led me to add those extra `---`. Thanks for pointing it out! I have reverted accordingly.